packages/smart-contracts/hardhat.config.ts

```diff
@@ -403,3 +404,14 @@ subtask(DEPLOYER_KEY_GUARD, 'prevent usage of the deployer master key').setActio
     throw new Error('The deployer master key should not be used for this action');
   }
 });
+
+task(
+  'update-fee-proxies-for-single-request-proxy-factory',
+  'Update the proxy addresses in SingleRequestProxyFactory',
+)
+  .addFlag('eoa', 'Is the update to be performed in an EOA context')
+  .setAction(async (args, hre) => {
+    const signWithEoa = args.eoa ?? false;
+    await hre.run(DEPLOYER_KEY_GUARD);
+    await updateFeeProxies(hre as HardhatRuntimeEnvironmentExtended, signWithEoa);
+  });
```

Comment:
Can you plug this into the existing 'update-contracts' task? 

Reference: 

- https://github.com/RequestNetwork/requestNetwork/blob/e91492cdc93b4ab3c3d7c5113e9dff225b4ba129/packages/smart-contracts/hardhat.config.ts#L372-L379
- https://github.com/RequestNetwork/requestNetwork/blob/e91492cdc93b4ab3c3d7c5113e9dff225b4ba129/packages/smart-contracts/scripts-create2/update-contracts-setup.ts#L9-L21
- https://github.com/RequestNetwork/requestNetwork/blob/e91492cdc93b4ab3c3d7c5113e9dff225b4ba129/packages/smart-contracts/scripts-create2/contract-setup/setups.ts#L17-L59
- https://github.com/RequestNetwork/requestNetwork/blob/5c83706a63dd6c6b6a34bcd6d2d7752fb825f596/packages/smart-contracts/scripts-create2/contract-setup/adminTasks.ts